src/getHumanAge.test.js

```diff
@@ -1,10 +1,60 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 for cat and 0 for dog if both inputs <15`, () => {
```

Comment:
Use single quotes instead of backticks for strings without template literals or variables. Replace all backticks in test descriptions with single quotes. 📝